src/ui/screens/SendFundScreen.tsx

```diff
@@ -26,15 +32,31 @@ export function SendFundScreen({navigation, route}: Props) {
   const {data: accountInfo} = useAccount(address);
   const ref = useRef<Modalize>(null);
   const [amount, setAmount] = React.useState('');
-  const [to, setTo] = React.useState<string>();
+  const [toAddress, setToAddress] = React.useState<string>();
   const [scanning, setScanning] = React.useState(false);
   const startTx = useApiTx();
   const {data: chainInfo} = useChainInfo();
-
+  const [isKeepAliveActive, setIsKeepAliveActive] = React.useState(true);
+  const {currentNetwork} = useContext(NetworkContext);
+  const snackbar = useSnackbar();
+  const {stringToBn} = useFormatBalance();
   useEffect(() => {
     ref.current?.open();
   }, []);
 
+  const isToAddressValid = useMemo(() => {
+    return toAddress ? isAddressValid(currentNetwork, toAddress) : false;
+  }, [toAddress, currentNetwork]);
+
+  const isEnteredBalanceValid = useMemo(() => {
+    const enteredBalance = stringToBn(amount) ?? BN_ZERO;
+    const isBalanceValid =
+      enteredBalance.gt(BN_ZERO) && enteredBalance.lt(formattedStringToBn(accountInfo?.balance.formattedFree));
+    return isBalanceValid;
+  }, [amount, accountInfo, stringToBn]);
+
+  const isSendDisabled = !isEnteredBalanceValid || !toAddress || !isToAddressValid;
```

Comment:
You can remove `!toAddress` here, `isToAddressValid` will be false if `toAddress` is `undefined`